pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "django-iranian-cities"
+version = "2.0.0"
+description = "Iranian cities support for Django"
+authors = ["Sepehr Akbarzadeh <sepehr@sageteam.org>"]
+license = "GNU"
+keywords = ["django", "iranian", "cities"]
+repository = "https://github.com/sageteamorg/django-iranian-cities"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
+django = ">=3.2,<6.0"
+
+[tool.poetry.dev-dependencies]
```

Comment:
make dev environment rich and standard:

[tool.poetry.group.dev.dependencies]
ruff
pre-commit
mypy
sphinx-rtd-theme
tox
pylint
docformatter
commitizen
django-stubs
pytest
pytest-django
pytest-cov
black
isort